tests/xharness/Jenkins.cs

```diff
@@ -874,6 +874,7 @@ public int Run ()
 					populating = false;
 				}).Wait ();
 				GenerateReport ();
+				BuildTestLibraries ();
 				if (!IsServerMode) {
 					foreach (var task in Tasks)
 						tasks.Add (task.RunAsync ());
```

Comment:
I might be wrong here, ProcessHelper.ExecuteCommandAsync does return a task, correct? 

```csharp
public static Task<bool> PollForExitAsync (int pid, TimeSpan timeout)
```

The task is ignored in BuildTestLibraries that returns void (which we should only do in events).  So, I think we should be paying attention to the result of the task and, since it is async but needed, we might wait for it to success before we do anything else (there is now guaranteed AFAIK that the build will be done before we needed it).

I might be completely wrong here, but just wanted you to confirm is correct.